src/rpsCommandLine.js

```diff
@@ -0,0 +1,158 @@
+import enquirer from "enquirer";
+
+class RpsCommandLine {
+  static WIN = 0;
+  static LOSE = 1;
+  static DRAW = 2;
+
+  async selectLevel() {
+    const response = await enquirer.prompt({
+      type: "select",
+      name: "level",
+      message: "CPUとのフラッシュじゃんけんです。\n 難易度を選択してください。",
+      choices: [
+        {
+          title: "Easy",
+          content: "全3問。\n1問当たり3.5秒表示されます。",
+          value: 3,
+        },
+        {
+          title: "Normal",
+          content: "全5問。\n1問当たり2.1秒表示されます。",
+          value: 5,
+        },
+        {
+          title: "Hard",
+          content: "全7問。\n1問当たり1.5秒表示されます。",
+          value: 7,
+        },
+      ],
+      footer() {
+        return `\n${this.focused.content}`;
+      },
+      result() {
+        return this.focused.value;
+      },
+    });
+    return response.level;
+  }
+
+  async selectRule() {
+    const displayRules = ["勝つ", "負ける", "同じ"];
+    const ruleIndex = Math.floor(Math.random() * 3);
+    process.stdout.write(
+      `CPUに対して${displayRules[ruleIndex]}手を選んでください`,
+    );
+
+    await this.#wait(2000);
+    process.stdout.clearLine();
+    process.stdout.cursorTo(0);
+    switch (ruleIndex) {
+      case RpsCommandLine.WIN:
+        return { [RpsCommandLine.WIN]: displayRules[ruleIndex] };
+      case RpsCommandLine.LOSE:
+        return { [RpsCommandLine.LOSE]: displayRules[ruleIndex] };
+      case RpsCommandLine.DRAW:
+        return { [RpsCommandLine.DRAW]: displayRules[ruleIndex] };
+    }
```

Comment:
`["勝つ", "負ける", "同じ"]`をstaticな定数にしてどこからでもアクセスできるようにしてあげれば、ここは

```suggestion
    return ruleIndex;
```

だけにできるのでは？